# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/crashreporter/nsDummyExceptionHandler.cpp
# Commit: 881add28e309
# Full Hash: 881add28e30909baaaa42c7eb290d48f23782767
# Author: Gabriele Svelto <gsvelto@mozilla.com>
# Date: 2022-09-12 21:40:55
# Regressor Bug: 1789421
# File Overlap Count: 1
# Description:
#   Bug 1789421 - Generate crash reports for content process shutdown hangs only when we can confirm that the process did not shut down in time r=jstutte
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D156838
# ==============================================================================

diff -r 41bbf2081f1b -r 881add28e309 toolkit/crashreporter/nsDummyExceptionHandler.cpp
--- a/toolkit/crashreporter/nsDummyExceptionHandler.cpp	Mon Sep 12 19:05:54 2022 +0000
+++ b/toolkit/crashreporter/nsDummyExceptionHandler.cpp	Mon Sep 12 19:05:24 2022 +0000
@@ -170,9 +170,13 @@
   return NS_ERROR_NOT_IMPLEMENTED;
 }
 
-void DeleteMinidumpFilesForID(const nsAString& id) {}
+void DeleteMinidumpFilesForID(const nsAString& aId,
+                              const Maybe<nsString>& aAdditionalMinidump) {}
 
-bool GetMinidumpForID(const nsAString& id, nsIFile** minidump) { return false; }
+bool GetMinidumpForID(const nsAString& id, nsIFile** minidump,
+                      const Maybe<nsString>& aAdditionalMinidump) {
+  return false;
+}
 
 bool GetIDFromMinidump(nsIFile* minidump, nsAString& id) { return false; }
 